docs/local_dev.rst

```diff
@@ -250,3 +250,20 @@ it is available in that local directory.
 
 To view the documentation open ``file://docs/build/html/index.html``
 with a web brower.
+
+Updating Test GeoIP Data and Libraries
+======================================
+The development environment uses a test MaxMind GeoIP database, and a test
+confirms that this is less than 1000 days old. To update this database and
```

Comment:
I know because I read the code that the test is part of Ichnaea, but I think this isn't clear in the text. I think I'd tweak it to "and an Ichnaea test" or "a test in the Ichnaea test suite" or something like that.